### DISCUSSION TITLE
3D Omni Lights cut in half after exporting to Web

### DISCUSSION THREAD

**Discussion by MerchantZoro** [godotengine/godot-proposals#10433] [3D]
v4.2.2

The Omni Lights get cut in half and the directional light also acts weird, I’m assuming because the browser renderer is different
How can I fix this?
![Captura de ecrã 2024-08-13 112956](https://github.com/user-attachments/assets/722c5676-8ac8-4b01-a736-b5b0d928b13e)

**Comment by AThousandShips** [comment]
This is for proposal discussions, bug reports are handled [here](https://github.com/godotengine/godot/issues), please search for existing issues and if you can't find any please make a new report

**Reply by MerchantZoro** [reply]
Thank you

**Reply by Calinou** [reply]
Issue opened: https://github.com/godotengine/godot/pull/60083
